layouts/shortcodes/protobuf.html

```diff
@@ -1,4 +1,8 @@
-{{ $protoName := .Get "name" }}
-{{ $protoDocsSummary := index .Site.Data.ProtoMap.apis $protoName }}
-{{ $protoDocsUrl := lower $protoDocsSummary.relativepath }}
-<a {{ printf "href=%v/%v" .Site.Data.Solo.DocsVersion $protoDocsUrl | safeHTMLAttr }}>{{ $protoName }}</a>
+{{- $protoName := .Get "name" -}}
+{{- $displayName := .Get "display" -}}
+{{- if eq $displayName "" -}}
+{{- $displayName = $protoName -}}
```

Comment:
these three lines can be simplified to:
```suggestion
{{- $displayName := .Get "display" | default $protoName -}}
```